docs/reference/modules/piece-page-type.md

```diff
@@ -0,0 +1,240 @@
+| [`pieceModuleName`](#pieceModuleName) | String | Optionally sets the `piece-type` to a specific name other than the default set by the module name. |
+| [`previous`](#previous) | Boolean \|\| Object | If set to `true`, `data.previous` is the previous piece based on the sort. |
+
+### `next`
+If this option is set to true, it exposes the next piece in the current [sort order](https://v3.docs.apostrophecms.org/reference/module-api/module-options.html#sort) as `req.data.next` (`data.next` in the template) when serving a [show page](https://v3.docs.apostrophecms.org/reference/glossary.html#general-terms). This can be used to provide a link to the next item in a series (e.g., the next oldest blog post).
+
+This option can also be set to an object containing an object with a key of `project` and values which will be used as a query builder for retrieving the next piece document.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  extend: ‘@apostrophecms/piece-page-type’,
+  options: {
+    next: true
+  },
+  // …
+}
+
+// OR
+
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    // The next article piece would be returned with only the `title, `_url`
+    // and `lastPublishedAt` properties.
+    next: {
+      project: {
+        title: 1,
+        _url: 1,
+        lastPublishedAt: 1
+      }
+    }
+  },
+  // …
+}
+```
+<template v-slot:caption>
+  modules/article-page/index.js
+</template>
+</AposCodeBlock>
+
+### `perPage`
+The `perPage` option receives an integer as value and sets the number of pieces that will be displayed per page for the `index.html` page. It is set to 10 items per page by default.
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    perPage: 20 // REST `GET` requests will return 20 pieces per page
+  },
+// …
+}
+```
+  <template v-slot:caption>
+    modules/article-page/index.js
+  </template>
+</AposCodeBlock>
+
+### `piecesFilter`
+The `piecesFilter` takes an array of objects to assist in filtering on the index page. Each object must have a `name` property associated with a valid [query builder](https://v3.docs.apostrophecms.org/reference/module-api/module-overview.html#queries-self-query). 
+
+These include:
+* Custom query builders configured in an app that include a `launder` method
+* Field names whose field types automatically get builders:
+    - boolean
+    - checkboxes
+    - date
+    - float
+    - integer
+    - relationship
+    - select
+    - slug
+    - string
+    - url
+
+When the index page is served, configured filters will be represented on a `req.data.piecesFilter` object (`data.piecesFilter` in the template). If you include `counts: true` in a filter object the number of pieces matching that filter are included in the `req.data.piecesFilter` properties.
```

Comment:
Spell out what the values inside `piecesFilter` will look like.

Reply:
Reworked this paragraph and added an example of what `data.piecesFilters` will look like.